common/djangoapps/course_modes/models.py

```diff
@@ -571,8 +590,11 @@ def enrollment_mode_display(cls, mode, verification_status):
 
         show_image = False
         image_alt = ''
+        enrollment_title = ''
+        enrollment_value = ''
+        display_mode = cls._enrollment_mode_display(mode, verification_status, course_id)
 
-        if mode == cls.VERIFIED:
+        if display_mode == "verified":
```

Comment:
mostly to be consistent with where the information is coming from.  `mode` was one of those constants, whereas `display_mode` was a string.  I suppose each of the `display_mode`s should probably be a constant as well though